pkg/storagev2/storagetest/benchmark.go

```diff
@@ -0,0 +1,297 @@
+package storagetest
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"math"
+	"math/rand"
+	"runtime"
+	"testing"
+	"time"
+
+	storage "github.com/ethersphere/bee/pkg/storagev2"
+)
+
+var keyLen int
+
+var (
+	valueSize        = flag.Int("value_size", 100, "Size of each value")
+	compressionRatio = flag.Float64("compression_ratio", 0.5, "")
+)
+
+const (
+	hitKeyFormat     = "%016d+"
+	missingKeyFormat = "%016d-"
+)
+
+func init() {
```

Comment:
I am confused about this `init` function, why is it needed? The `hitKeyFormat` and `missingKeyFormat` are constants and so is the `math.MaxInt32`, doesn't it lead to the same result (17) every time the `init` is called?